coriolis/tests/api/v1/data/minion_pool_exceptions.yml

```diff
@@ -0,0 +1,5 @@
+- config:
```

Comment:
This `config` is practically empty. I think you could just pass the exception names directly:
```yml
- NotFound
- InvalidParameterValue
```